docs/_steps/saving_workflow_step.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Saving the step configuration
+lang: en
+slug: saving-steps
+order: 4
+---
+
+<div class='section-content'>
+
+When the workflow step's configuration has been saved (using the step configuration modal from the `edit` callback), your app will listen for the `view_submission` event. The method assigned to the `save` property of the `WorkflowStep` configuration object passed in during instantiation will run when this event occurs.
+
+Once the configuration for the workflow step has been determined, builders often use that configuration to craft the custom outputs and behavior that occurs when the end user executes the step.
+
+Within the `save` callback, the `update()` method can be used to save the builder's step configuration by passing in the following arguments: `inputs`, `outputs`, `step_name` and `step_image_url`.
+
+`inputs` is an object representing the data your app expects to receive from the user upon workflow step execution. To use variables that were collected earlier in the workflow, you can include handlebar-style syntax (`{{ variable }}`). During the workflow step's execution, those variables will be replaced with their actual runtime value.
```

Comment:
You could make these a list to make them a little bit more readable. I think they're the most important content, so it'd be nice to make them glanceable.

Reply:
Also, when listing them, I would suggest bolding the property names (like **`inputs`**)